rosdep/base.yaml

```
@@ -6846,6 +6846,7 @@ matio:
   rhel:
     '*': null
     '8': [matio-devel]
+    '9': [matio-devel]
```

Given that we expect this package to be available moving forward, I think we should redo this a bit to be:

```suggestion
  rhel: [matio-devel]
```